01 - Community/Authors - Persons/AndreasStandar.md

```diff
@@ -0,0 +1,57 @@
+---
+aliases:
+- <Serice
```

Comment:
Is the starting `<` intentional?

Reply:
That comes from their own theme/plugin info I believe.

Reply:
Yep, see: https://github.com/obsidianmd/obsidian-releases/pull/621/files